src/pages/products/Products.jsx

```diff
@@ -0,0 +1,6 @@
+import "./products.css";
+
+export function Products() {
+  document.title = "Products - SmashCart - Manoj Sarna";
```

Comment:
Suggestion: move document.title in useEffect.
Reason: right now with this approach title is being set at each re-render. This can be avoided by making it render only once in useEffect.